beeai/agents/backport_agent.py

```diff
@@ -70,6 +76,32 @@ def input_schema(self) -> type[TInputSchema]:
     def output_schema(self) -> type[TOutputSchema]:
         return OutputSchema
 
+    def _render_prompt(self, input: TInputSchema) -> str:
+        # Define template function that can be called from the template
+        def backport_git_steps(data: dict) -> str:
+            input_data = self.input_schema.model_validate(data)
+            return get_backport_git_steps(
+                package=input_data.package,
+                jira_issue=input_data.jira_issue,
+                git_url=input_data.git_url,
+                gitlab_user=input_data.gitlab_user,
+                git_user=input_data.git_user,
+                git_email=input_data.git_email,
+                dist_git_branch=input_data.dist_git_branch,
+                dry_run=input_data.dry_run
+            )
```

Comment:
Sorry, maybe I'm missing something, but I don't understand why there has to be an extra layer (and translation of arguments). Can't we call `get_git_finalization_steps()` directly from here?

Also, the finalization steps could be a `PromptTemplate` and the input arguments a model/schema, but I'm fine with the implementation as it is.

Reply:
ok, removed and moved the helper method to utils.py since its just one function now